index.js

```diff
@@ -11,8 +11,8 @@ function format(obj) {
     : JSON.stringify(obj);
 }
 
-module.exports = function(hash) {
-  function visit(hash, prefix) {
+module.exports = function(hash, indent=0, newline_after_section=false) {
+  function visit(hash, prefix, indent, newline_after_section) {
```

Comment:
If `indent` and `newline_after_section` never change for `visit()` then perhaps they don't need to be parameters here; instead. `visit()` can just use the values passed to the parent function.

Reply:
Done.